.github/workflows/go-migrate.yml

```diff
@@ -0,0 +1,65 @@
+# This workflow is for validating Rill migration files.
+
+name: Validate Migrations
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/go-migrate.yml"
+      - "runtime/drivers/sqlite/migrations/**"
+      - "admin/database/postgres/migrations/**"
+  workflow_dispatch:
```

Comment:
The main idea here is actually to test on the main/release branch merges, not on PRs.

The problem we're trying to mitigate is folks cherry-picking a patch into a release branch which ends up skipping a migration from another PR that was not cherry-picked.